### ISSUE TITLE
DECSYSTEM 2020 RH11-TM02/03-TU45/TU77 BASIC DEVICE DIAGNOSTIC (DSTUA) LRCC Character Failure

### ISSUE THREAD

**Issue by jfcl** [KS10FPGA/KS10FPGA#24]
DSTUA TST114 subtest 2 and TST117 subtest 2 fail Longitudinal Redundancy Check Character (LRCC or LRC) tests.

Physically, the 9-track tape format provides vertical parity across the tape; i.e., 8 data bits and 1 parity bit.  The tape formatter/controller also provides longitudinal parity such that at the end of the record, a longitudinal redundancy check bit is written in any track if the longitudinal count is odd.

Because the SIMH tape file format is lossless and does not provide parity bits, adding parity calculation to the tape interface just to pass this test provides dubious benefits.

```
*** TST # 114 ; TM03/TU77 0/0 ; ERR PC = 35771
*** SIMPLE MOTION AND DATA XFERS - NRZ

LRC CHARACTER (FROM MAINT REG) WRONG
GOOD: 667  BAD: 000  XOR: 667
POSSIBLE BOARDS:  M8934/5, M8905/6,

ERROR WAS DURING SUBTEST 2
```

and

```
*** TST # 117 ; TM03/TU77 0/0 ; ERR PC = 35771
*** SIMPLE MOTION AND DATA XFERS - NRZ

LRC CHARACTER (FROM MAINT REG) WRONG
GOOD: 665  BAD: 000  XOR: 665
POSSIBLE BOARDS:  M8934/5, M8905/6,

ERROR WAS DURING SUBTEST 2
```

**Comment by jfcl** [comment]
I probably won't fix this.  This condition can never occur except when executing diagnostic software.  SIMH doesn't implement this either.

**Comment by jfcl** [comment]
It was simple enough to fix.
Fixed by commit 8b16f263238c34affa69e7c60ce2a09c965352c7
